README.md

```diff
@@ -47,9 +47,26 @@ it('should sign in', function (done) {
     .expect(200)
     .end(done);
 });
+```
+
+You can set preconditions:
+
+```js
+describe('after authenticating session', function () {
+  beforeEach(function (done) {
+    testSession.post('/signin')
+      .send({ username: 'foo', password: 'password' })
+      .expect(200)
+      .end(function (err) {
+        if (err) return done(err);
+        authenticatedSession = testSession;
+        return done();
+      });
+  });
+});
 
 it('should get a restricted page', function (done) {
-  testSession.get('/restricted')
+  authenticatedSession.get('/restricted')
```

Comment:
I think this still leaves us in a broken state, though, as `authenticatedSession` isn't available here. But we can:

  1. Move this test inside the `'after authenticating'` spec
  2. Define `var authenticatedSession`in the top level of the spec